examples/tfengine/generated/full/live/prod/team1/example-prod-apps/project/terragrunt.hcl

```diff
@@ -29,5 +30,6 @@ dependency "networks" {
 }
 
 inputs = {
-  folder_id = dependency.parent_folder.outputs.name
+  folder_id = "${dependency.parent_folder.outputs.name}"
```

Comment:
Is this correct? I've previously seen it be like it was before, without the `${}`.

Reply:
I changed the template to use the new `hcl` func in funcmap, that's why this is now needed. Before its value was just set directly.

Reply:
I see. How come terragrunt can read this kind of syntax though, in the generated files?

Reply:
They added support for it.

https://github.com/gruntwork-io/terragrunt/blob/master/config/config.go#L58
https://github.com/gruntwork-io/terragrunt/blob/master/config/config.go#L349

Reply:
I see.